zcnbridge/config.go

```diff
@@ -221,22 +221,31 @@ func (b *BridgeOwner) ClientID() string {
 
 // SetupBridgeClientSDK Use this from standalone application
 // 0Chain SDK initialization is required
-func SetupBridgeClientSDK(cfg *BridgeSDKConfig) *BridgeClient {
+func SetupBridgeClientSDK(cfg *BridgeSDKConfig, walletFile string) *BridgeClient {
 	log.InitLogging(*cfg.Development, *cfg.LogPath, *cfg.LogLevel)
-	bridgeClient := CreateBridgeClient(initBridgeConfig(cfg))
+	bridgeClient := CreateBridgeClient(initBridgeConfig(cfg), walletFile)
 	return bridgeClient
 }
 
 // SetupBridgeOwnerSDK Use this from standalone application to initialize bridge owner.
 // 0Chain SDK initialization is not required in this case
-func SetupBridgeOwnerSDK(cfg *BridgeSDKConfig) *BridgeOwner {
+func SetupBridgeOwnerSDK(cfg *BridgeSDKConfig, walletFile string) *BridgeOwner {
 	log.InitLogging(*cfg.Development, *cfg.LogPath, *cfg.LogLevel)
-	bridgeOwner := CreateBridgeOwner(initBridgeConfig(cfg))
+	bridgeOwner := CreateBridgeOwner(initBridgeConfig(cfg), walletFile)
 	return bridgeOwner
 }
 
-func loadWallet(homedir string) (*zcncrypto.Wallet, error) {
-	walletPath := path.Join(homedir, ZChainWalletConfigName)
+func loadWallet(homedir, fileName string) (*zcncrypto.Wallet, error) {
+	var walletPath string
+
+	if fileName != "" {
+		walletPath = path.Join(homedir, fileName)
+	} else {
+		walletPath = path.Join(homedir, ZChainWalletConfigName)
+	}
+
+	fmt.Println(walletPath)
```

Comment:
Avoid print logs in sdk APIs or print with the `log` initialized so that the logs could be turned off when needed.

Reply:
Forgot to remove